src/app/pages/login/login.component.html

```diff
@@ -3,7 +3,8 @@
     <div class="header-body text-center mb-7">
       <div class="row justify-content-center">
         <div class="col-lg-5 col-md-6">
-          <h1 class="text-white">I Stand With Health Professionals.?</h1>
+          <!--We can improve the grammer by adding "Do you"-->
```

Comment:
Yeah, can you please add the suggestion with Do you as you noted? Otherwise it looks good.